javascript-javascript1-week4/voiceAssistant.js

```diff
@@ -0,0 +1,175 @@
+let nameOfPerson = "";
+let toDoList = [];
+
+const commands = {
+  "Hello my name is": welcomeWithName,
+  "What is my name": getName,
+  Add: addToDo,
+  Remove: removeToDo,
+  "What is on my todo?": getToDo,
+  "What day is it today?": getDate,
+  "What is": getResult,
+  "Set a timer for": setTimer,
+  "Tell me a evening jokes": tellJokes,
+};
+
+function welcomeWithName(command) {
+  const userName = command.replace("Hello my name is", "").trim();
```

Comment:
What would happen if a user types `hello my name is Valentin`?

Reply:
Thank you for the suggestion, i will work on that.